shared/fs/row/tlf.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import * as React from 'react'
+import {globalColors, isMobile} from '../../styles'
+import {rowStyles, StillCommon, type StillCommonProps} from './styles'
+import {Box, Box2, Meta, Text} from '../../common-adapters'
+import PathItemInfo from '../common/path-item-info'
+
+type TlfProps = StillCommonProps & {
+  isIgnored: boolean,
+  isNew: boolean,
+  isUserReset: boolean,
+  needsRekey: boolean,
+  resetParticipants: Array<string>,
+}
+
+const RowMeta = ({isNew, isIgnored, needsRekey}) => {
+  if (isIgnored || !(isNew || needsRekey)) {
+    return null
+  }
+
+  return (
+    <Box style={{width: 0, display: 'flex'}}>
```

Comment:
what is this box for

Reply:
also this likely doens't work on android

Reply:
It was this structure in the previous version already. I'm reluctant to change it in this ticket, but I'm happy to try changing it later next time we need to touch it.